### PR TITLE
Add etlegacy v2.77.1

### PR DESCRIPTION
**Pull request by sebdanielsson**
- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.
- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

Second cask so please correct any errors you may encounter.
- wget dependency is because the version of curl that macOS is shipping with doesn't allow the --output-dir option.
- Running wget 3 times because the command `wget https://mirror.etlegacy.com/etmain/pak{0..2}.pk3 -P etlegacy/etmain` don't work in Ruby that I know of. (Not a programmer tho)

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Neat trick! But we won’t accept it as is. No other cask uses this method and adding something different for what will be a niche cask isn’t a good idea.

If you can make it work with our regular methods, we’ll reopen and reconsider. Otherwise, you may wish to host on your own tap.

Tips: don’t download anything manually (i.e. outside of `url`); don’t install dependencies which are only useful for the installation process.

Thank you anyway.

**Comment by SebDanielsson** [comment]
I'll try to figure something out :)

Any tips? Can one use multiple "url"?

File: Casks/etlegacy.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cask "etlegacy" do
+  version "2.77.1"
+  sha256 :no_check
+
+  url "https://www.etlegacy.com/download/file/258"
+  name "ET: Legacy"
+  desc "Open source project based on the code of Wolfenstein: Enemy Territory"
```
Comment (vitorgalvao) [inline Casks/etlegacy.rb:7]: `Game based on the code of Wolfenstein: Enemy Territory`

**Comment by vitorgalvao** [comment]
> Any tips?

Not without investigating the software further.

> Would it be okay if I dropped the wget dep and tried to use a curl command to get the files?

No. That falls into my tip above.

> would it be okay if the etlegacy projekt offered a .sh script that I could call on to download the files?

Yes, as long as you get it with `url` and it does everything. That’s [`installer script:`](https://docs.brew.sh/Cask-Cookbook#installer-script).

**Pull request closed without merging**